### PR TITLE
Corrected the documentations for the fluid viscosity

### PR DESCRIPTION
**Pull request by wenqing**
As titled.

### CODE REVIEW

File: MaterialLib/Fluid/Viscosity/createViscosityModel.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace Fluid
 static std::unique_ptr<FluidProperty> createLinearPressureDependentViscosity(
```
Comment (wenqing) [inline MaterialLib/Fluid/Viscosity/createViscosityModel.cpp]: The first character of this file name and that in createDensityModel.cpp will be changed to its capital one, "C".

File: MaterialLib/Fluid/Viscosity/createViscosityModel.cpp [hunk 2]
```diff
@@ -32,7 +32,7 @@ namespace Fluid
...
     BaseLib::ConfigTree const& config)
 {
-    //! \ogs_file_param{material__fluid__viscosity__type}
+    //! \ogs_file_param{material__fluid__viscosity__LinearPressure}
```
Comment (endJunction) [inline MaterialLib/Fluid/Viscosity/createViscosityModel.cpp]: In ogs_file_param{xxx} the xxx refers to the first argument to the checkConfigParameter() call. So in this case the original version with '__type' is correct. Only the subsequent references in ogs_file_param include the specific case , e.g. LinearPressure.

**Comment by endJunction** [comment]
Also have a look on the generated doxygen documentation. The issues shown below should not be added.
image

**Comment by wenqing** [comment]
@endJunction  After our discussion, I modified this PR to documentation the unreferenced input parameters of viscosity types and density types.

**Pull request merged**